src/gaggiuino.ino

```diff
@@ -22,6 +22,7 @@
 #define DESCALE_PHASE1_EVERY    500 // short pump pulses during descale
 #define DESCALE_PHASE2_EVERY    5000 // short pause for pulse effficience activation
 #define DESCALE_PHASE3_EVERY    120000 // long pause for scale softening
+#define TEMP_DELTA              setPoint*0.25f
```

Comment:
Where is setPoint coming from here? I suspect this wouldn't work unless you make setPoint a #defined variable too?